toolkit/tools/scheduler/buildagents/definition.go

```diff
@@ -9,13 +9,14 @@ import "fmt"
 type BuildAgentConfig struct {
 	Program string
 
-	WorkDir   string
-	WorkerTar string
-	RepoFile  string
-	RpmDir    string
-	SrpmDir   string
-	CacheDir  string
-	CCacheDir string
+	WorkDir      string
+	WorkerTar    string
+	RepoFile     string
+	RpmDir       string
+	ToolchainDir string
```

Comment:
Is `ToolchainDir` used for anything?

Reply:
Its passed to the build agent via `serializeChrootBuildAgentConfig()` (chrootagent.go:84)

Reply:
Yes, but keeping it, just so we can print it seems pointless. And I think it's not used by the code in any other way.

Reply:
Sorry, I didn't clarify fully. It gets passed to the serializer, which passes that flag to the build agents so they can mount the toolchain directory and install RPMs from it during a build.

It gets passed through to `pkgworker.go:158` where it is used to create an overlay.